### ISSUE TITLE
Revision information in testcases is often not present and inconsistent

### ISSUE THREAD

**Issue by mikea** [google/oss-fuzz#273]
Consider https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=396

- The issue itself should mention the latest revision that was used to reproduce the build (developers want to be certain it is reproducible)

- the testcase shows 0:<first_revision> as revision range. Which also hides the fact that issue is still reproducible in HEAD.

I suggest:

- add crash revision to issue text
- instead of regression revision display "reproducible revisions". Thus it would be clear when issues were reproduced.

**Comment by inferno-chromium** [comment]
We fixed stuff and now show three things now properly, e.g. checkout 
https://clusterfuzz-external.appspot.com/v2/testcase-detail/5372912110403584

1)
Fixed Section:
NO (Last reproducible crash with revision e3bc1305ec97d4ad4ab05fa59a288e92df2b2025)
This helps to show bug is still there and what was the last revision it was tested with.

2) 
ORIGINAL STACKTRACE ON REVISION 1ED327F7DFE02A062E50CD844A925C970C645C7C
and its stacktrace

3)
LAST TESTED STACKTRACE ON REVISION E3BC1305EC97D4AD4AB05FA59A288E92DF2B2025
and its stacktrace

We cannot substitute regression range for reproducible revision range. Regression range is really important for developers to know when a regression was introduced and we cannot get rid of that.

Regarding putting crash revision in issue text, we are already filing up the issue with too much information. I discussed this with Oliver as well and he also thinks this is already lot of info in issue itself. developers should go to clusterfuzz report anyway to see stacktrace, etc. so it makes sense to just keep it there.
